index.js

```diff
@@ -13,8 +13,8 @@ module.exports = {
   name: 'ember-d3',
 
   allD3Modules(target) {
-    let deps = target.dependencies()
-    let d3ModuleNames = Object.keys(deps).filter(dep => dep.startsWith('d3'))
+    let deps = Object.values(target.project.packageInfoCache.entries).map(obj => obj.name || '')
```

Comment:
@jrjohnson i don't think that cache is public API...

Reply:
I found it here: https://ember-cli.com/api/classes/PackageInfoCache.html I thought this was only public stuff, but maybe I'm wrong? You would certainly know better than I would.  Is there another way to get the dependency tree of an app which includes all addons?

Reply:
https://ember-cli.com/api/classes/Project.html does not mention `packageInfoCache` as public API though, and https://ember-cli.com/api/classes/PackageInfoCache.html is not listing `entries` 😉 

>  Is there another way to get the dependency tree of an app which includes all addons?

Not that I'm aware of, other than iterating the `package.json` files yourself 😞

Reply:
Ah ok. Bad use then.  Sorry.  Interstingly it doesn't look like https://ember-cli.com/api/classes/Project.html#method_dependencies was public either. I'm went ahead and opened https://github.com/ember-cli/rfcs/issues/123 since I think this warrants `ember-cli` discussion.